api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1541,6 +1541,12 @@ public ObjectStat statObject(String bucketName, String objectName, ServerSideEnc
            InvalidKeyException, NoResponseException, XmlPullParserException, ErrorResponseException,
            InternalException, InvalidArgumentException, InvalidResponseException {
     checkReadRequestSse(sse);
+    checkBucketName(bucketName);
+    if ( objectName == null ) {
+      throw new InvalidArgumentException("Invalid Stat Object Argument(s). Object name cannot be null.");
+    } else if ( objectName.equals("") ) {
```

Comment:
1. `else` is not needed here.
2. please fix ci failures.

Reply:
1. Else removed.
2. CI failures are not related to this change. We are debugging the root-cause of it.